db/fixtures/miq_product_features.yml

```diff
@@ -605,6 +605,15 @@
       :description: Edit Tags of Object Store Containers
       :feature_type: control
       :identifier: cloud_object_store_container_tag
+  - :name: Modify
+    :description: Modify Object Store Containers
+    :feature_type: admin
+    :identifier: cloud_object_store_container_admin
+    :children:
+    - :name: Remove
```

Comment:
remove, delete, destroy 😆  

But here you are following using `Remove` for `_delete` ops